### DISCUSSION TITLE
本地部署是如何自定义输出端口？

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#1321] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/1217096/187053459-68b59e78-dc3f-4e31-b24b-1ee4d0b2fa34.png)

 本地部署是使用的端口是 `8360` ， 这个可以在启动时增加动态参数调整吗？

注：发现 `production.json` 有个 `port` 参数，只能这里调整吗？

**Comment by lizheming** [comment, marked as answer]
```js
// index.js
module.exports = Waline({
  port: 3000,
})
```

或者 `node node_modules/@waline/vercel/vanilla.js 3000` 试试。

**Reply by elkan1788** [reply]
直接使用node命令成功了，谢谢
